Clothelate 2

kokolo21 opened the pull request:
<!-- Escribe **ABAJO** de los encabezados y **ARRIBA** de los comentarios de otra manera, podria terminar sin verse correctamente el texto -->

## Que hace este PR
<!-- Añade una buen parrafo de todo lo que hace este PR, en caso de no hacerlo retrasaras el proceso de review de tu PR -->
Agrega la clothelate al juego, una autolathe que solo crea ropas apartir de metal y requiere ser creada con research de R&D de ilegal. 
## Por que es bueno este PR
<!-- Añade una explicacion para los cambios que hicistes, si no puedes explicar por que darle un .50 a un payaso es una buena idea, lo mas probable es que no valga la pena añadirlo -->
Ya no va a faltar ropas siempre que haya metal. Ademas, se pueden conseguir disfraces con esto y se puede agregar facilmente mas cosas.

## Imagenes de Cambios
<!-- Si no tienes ningun cambio en un mapa o sprite, o mecanica puedes borrar esta seccion. Aun asi, considera hacer mp4 o gifs de features -->

kokolo21 committed:
Clothelate (WIP)

WIP

Inicios de la clothelate y ver si se puede hacer andar el tgui

kokolo21 committed:
Testing TGUI

kokolo21 committed:
Arreglos e interfaz funcional

Ya la interfaz funciona como debe ser y ahora falta agregar la ropa.

kokolo21 committed:
arreglos

kokolo21 committed:
update de designs

kokolo21 committed:
arreglos

AsrielFBI committed:
Las ropas uwu

kokolo21 committed:
Merge pull request #1 from AsrielFBI/pr/1156

Las ropas uwu

kokolo21 committed:
Revert "update de designs"

This reverts commit e1bdc0766f23320b0283adab304eae71e7eca250.

kokolo21 committed:
Merge branch 'Clothelate' of https://github.com/kokolo21/Paradise into Clothelate

kokolo21 committed:
fixing an oopsie

kokolo21 committed:
fixing oopsies

kokolo21 committed:
Merge branch 'Lambda' into Clothelate

kokolo21 committed:
Merge branch 'Clothelate' into Clothelate2

kokolo21 committed:
Arreglos

kokolo21 commented:
![imagen](https://media.discordapp.net/attachments/586307829087076362/923044761374441492/clothelate-Suit.gif)

AsrielFBI reviewed (changes requested):
Por solicitud de un admin. Baja que se requiera Illegal Research 3 a 2.

kokolo21 committed:
Illegal de 3 a 2 de requisito

A pedido de admin, el coste illegal cambia de 3 a 2 para que sea mas facil de hacer.

AsrielFBI commented:
El error que hay es para el unit test de delta. Cosa que nunca fue modificado y siempre falla en muchos PRs que aprueban de igual forma. Así que pa' lante.
